flang/lib/Semantics/check-omp-structure.cpp

```diff
@@ -383,6 +383,21 @@ void OmpStructureChecker::CheckMultListItems() {
     CheckMultipleOccurrence(
         listVars, nontempNameList, clause->source, "NONTEMPORAL");
   }
+
+  // Linear clause
+  for (auto [_, clause] : FindClauses(llvm::omp::Clause::OMPC_linear)) {
+    const auto &linearClause{std::get<parser::OmpClause::Linear>(clause->u)};
+    std::list<parser::Name> nameList;
+    common::visit(
+        common::visitors{
+            [&](const auto &u) {
+              std::copy(
+                  u.names.begin(), u.names.end(), std::back_inserter(nameList));
+            },
+        },
```

Comment:
You only have one visitor, so you don't need `common::visitors`.  Do `common::visit([&]...)`.

Reply:
Yea, sure.